packages/react-core/src/components/Wizard/WizardFooterInternal.tsx

```diff
@@ -32,7 +32,12 @@ export const WizardFooterInternal: React.FunctionComponent<WizardFooterInternalP
       {nextButtonText}
     </Button>
     {!activeStep.hideBackButton && (
-      <Button variant={ButtonVariant.secondary} onClick={onBack} className={css(firstStep && 'pf-m-disabled')}>
+      <Button
+        variant={ButtonVariant.secondary}
+        onClick={onBack}
+        className={css(firstStep && 'pf-m-disabled')}
+        isDisabled={firstStep ? true : null}
```

Comment:
className can be removed, 'pf-m-disabled' will be added by the isDisabled property now.

Reply:
Oh nice catch!! I didn't even see that. 😄